src/parser/jobs/gnb/modules/SonicBreak.tsx

```diff
@@ -0,0 +1,175 @@
+			if (event.targets.find(e => e.amount !== 0) && event.cause.status === this.data.statuses.SONIC_BREAK.id) { //Dot did damage, increment.
+				this.breakUpdater(event)
+			}
+		}
+	}
+
+	private onDotRemove(event: Events['statusRemove']) {
+		this.breakEnder(event)
+	}
+
+	private breakMaker(event: Event) : void {
+		const sonicBreakWindow = new SonicBreakApplication(event.timestamp)
+		this.SonicBreakApplications.push(sonicBreakWindow)
+	}
+
+	private breakUpdater(event: Event) : void {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		//We missed the break maker somehow and now we need to make it.
+		if (lastSonicBreak == null || lastSonicBreak.end != null) {
+			this.breakMaker(event)
+		}
+
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+			lastSonicBreak.totalTicks++
+		}
+	}
+
+	private breakEnder(event: Event) : void {
+		const lastSonicBreak = this.lastSonicBreakApplication
+
+		if (lastSonicBreak != null && lastSonicBreak.end == null) {
+
+			lastSonicBreak.end = event.timestamp
+
+			if (lastSonicBreak.totalTicks < MAX_TICKS) {
+				lastSonicBreak.isMissingTicks = true
+			}
+		}
+	}
+
+	private onComplete() {
+		const badSonicBreaks = this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks)
+		if (badSonicBreaks.length > 0) {
+			const missedTicks = badSonicBreaks.reduce((acc, val) => acc + (MAX_TICKS - val.totalTicks), 0)
+			this.suggestions.add(new TieredSuggestion({
+				icon: this.data.actions.SONIC_BREAK.icon,
+				content: <Trans id="gnb.sonic-break.suggestions.missing-ticks.content">
+					One or more of your <DataLink action = "SONIC_BREAK"/> dots had ticks that did not deal damage. </Trans>,
+				tiers: {
+					1: SEVERITY.MINOR,
+					5: SEVERITY.MEDIUM,
+					10: SEVERITY.MAJOR,
+
+				},
+				value: missedTicks,
+				why: <Trans id = "gnb.sonic-break.suggestions.missing-ticks.why"> You wasted {missedTicks} ticks of <DataLink action = "SONIC_BREAK"/> </Trans>,
+			},
+			))
+		}
+	}
+
+	override output() {
+		if (this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks).length === 0) { return }
+
+		const missedTickWindows = this.SonicBreakApplications.filter(sonicBreak => sonicBreak.isMissingTicks)
+
+		const tickTable = <Table collapsing unstackable compact="very">
+			<Table.Header>
+				<Table.Row>
+					<Table.HeaderCell><Trans id="gnb.sonic-break.ticktable">Timestamp</Trans></Table.HeaderCell>
+					<Table.HeaderCell><Trans id="gnb.sonic-break.ticktable.drift-header">Missing Ticks</Trans></Table.HeaderCell>
+					<Table.HeaderCell></Table.HeaderCell>
+				</Table.Row>
+			</Table.Header>
+			<Table.Body>
+				{missedTickWindows.map(window => {
+					return <Table.Row key={window.start}>
+						<Table.Cell>{this.parser.formatEpochTimestamp(window.start)}</Table.Cell>
+						<Table.Cell>
+							<Trans id="sonic-break.tick-issue">
+								<DataLink action = "SONIC_BREAK"/> had {MAX_TICKS - window.totalTicks} ticks deal no damage.
```

Comment:
_i18n:_ `{MAX_TICKS - window.totalTicks} ticks` should be made into an embedded plural.